datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/SchemaTable.tsx

```diff
@@ -41,15 +43,20 @@ export type Props = {
     editableSchemaMetadata?: EditableSchemaMetadata | null;
     editMode?: boolean;
     usageStats?: UsageQueryResult | null;
+    getSchemaBlameResult?: GetSchemaBlameResult | null;
```

Comment:
It appers that we only use "schemaFieldBlameList" inside of this component.

Considering changing this prop to take an Array<SchemaFieldBlame> instead of the higher level graphql result object.

Reply:
Agreed!